apps/api/src/app/auth/auth.controller.ts

```diff
@@ -59,13 +61,19 @@ export class AuthController {
   ) {}
 
   @Get('/github')
-  githubAuth() {
+  githubAuth(context: ExecutionContext) {
+    Logger.verbose('Checking Github Auth');
```

Comment:
Do we have any metadata on the log that will be added to those logs?  for example transactionId/ environmentId/ userId etc...

Reply:
Good point! I think that we should add the metadata for : environmentId, userId, organizationId whenever a jwt token is available in the request, @Cliftonz how can we do that?